### PR TITLE
XUI MO enable HPA

### PR DESCRIPTION
**Pull request by ernestman28**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3068

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "@types/global-agent": "^2.1.0",
     "applicationinsights": "^1.5.0",
     "applicationinsights-js": "^1.0.20",
-    "axios": "^0.18.0",
+    "axios": ">=0.21.1",
```
Comment (paul-graham) [inline package.json:83]: Why is this `>=0.21.1` while the other projects have it as `^0.21.1`?

Comment (leepavlouhmcts) [reply package.json:83]: there were some high vulnerabilities. after running 'yarn audit --groups dependencies' it suggested >=0.21.1. we confirmed with uday the version number. he is adding the same on his AO feature branch.

Comment (paul-graham) [reply package.json:83]: Understood. Then we should presumably do the same on MC, MO, and any other projects that use Axios.

**Review by paul-graham — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]
LGTMl

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Pull request merged**